templates/react-native/README.md.twig

```diff
@@ -0,0 +1,42 @@
+# {{ spec.title }} {{sdk.name}} SDK
+
+![License](https://img.shields.io/github/license/{{ sdk.gitUserName|url_encode }}/{{ sdk.gitRepoName|url_encode }}.svg?style=flat-square)
+![Version](https://img.shields.io/badge/api%20version-{{ spec.version|url_encode }}-blue.svg?style=flat-square)
+[![Build Status](https://img.shields.io/travis/com/appwrite/sdk-generator?style=flat-square)](https://travis-ci.com/appwrite/sdk-generator)
+{% if sdk.twitterHandle %}
+[![Twitter Account](https://img.shields.io/twitter/follow/{{ sdk.twitterHandle }}?color=00acee&label=twitter&style=flat-square)](https://twitter.com/{{ sdk.twitterHandle }})
+{% endif %}
+{% if sdk.discordChannel %}
+[![Discord](https://img.shields.io/discord/{{ sdk.discordChannel }}?label=discord&style=flat-square)]({{ sdk.discordUrl }})
+{% endif %}
+{% if sdk.warning %}
+
+{{ sdk.warning|raw }}
+{% endif %}
+
+{{ sdk.description }}
+
+{% if sdk.logo %}
+![{{ spec.title }}]({{ sdk.logo }})
+{% endif %}
+
+## Installation
+
+To install
+
+```bash
+npm install react-native-appwrite react-native-fs react-native-url-polyfill --save
+```
+
```

Comment:
Do we need
``````
For iOS apps, you will need to install cocoapod dependencies.

```sh
cd iOS && pod install && cd ..
```
``````

Reply:
We have it in the `getting started section`